src/app/components/Dialog/Dialog.tsx

```diff
@@ -27,65 +30,15 @@ function Dialog(props: {
           position: 'absolute',
           left: '50%',
           top: '50%',
-          width: '500px',
-          height: '300px',
-          margin: '-150px 0 0 -250px',
+          width: '30rem',
+          height: '20rem',
+          borderRadius: '0.8rem',
+          margin: '-10rem 0 0 -15rem',
```

Comment:
width, height 는 prop으로 받을 수 있었으면 좋겠습니다.
그리고 정중앙에 놓을때 마이너스 margin 말고 translate로 하면 크기와 상관없이 가운데 놓을 수 있어요.

```suggestion
          width: props.width,
          height: props.height,
          borderRadius: '0.8rem',
          transform: translate(-50%, -50%),
```

Reply:
수정했습니다!